internal/lib/checkpoint.go

```diff
@@ -164,6 +162,9 @@ func (c *ContainerServer) prepareCheckpointExport(ctr *oci.Container) error {
 			}
 			return c.config.DefaultRuntime
 		}(),
+		CheckpointedAt: time.Now(),
+		RestoredAt:     ctr.RestoredAt(),
+		Restored:       ctr.Restore(),
```

Comment:
I'm confused, why would there be Restored/RestoredAt if it's checkpointed here? is this for previous checkpoint/restore cycles?

Reply:
CheckpointedAt will always be the time of last checkpoint. Either when the current container was checkpointed or if the current container has not been checkpointed but restored from a checkpoint archive it will be time of the last checkpoint on a possible other system.

RestoredAt will be the time of the last restore. This field is kind of unnecessary because if `Restored` is true it will be the same time as `startedAt`. It gives information about that this is restored container and when it was restored.

One could argue that having both CheckpointedAt and Restored is unnecessary, but a container could have been checkpointed but not restored and then Restored would be false.

Restore and RestoredAt in combination with `startedAt` could be simplified by removing RestoredAt as it does not carry any additional information. It is just easier to see when it was restored.

Reply:
yeah I guess my feeling is it wouldn't hurt to simplify the metadata, as it would reduce code complexity and the tiny amount of overhead it takes to transmit. if you can glean the information needed from less, it woulid be my preference to go that route. WDYT?

Reply:
removed `restoredAt`